generate_route.py

```diff
@@ -16,27 +14,23 @@
 
 """Generate a route that can be made into a User Motion File based on two locations.
 
-Leave one blank line.  The rest of this docstring should contain an
-overall description of the module or program.  Optionally, it may also
-contain a brief description of exported classes and functions and/or usage
-examples.
+Classes for Location, Route, and TimedRoute. A Route is a list of points that connect
+a user given start and end location. TimedRoute is a child class of Route and incorporates
+a user specified speed of travel and point frequency, which can be used to
+create a user motion file (10 Hz) with various simulated speeds (walking, running, biking)
 
   Typical usage example:
-
-  foo = ClassFoo()
-  bar = foo.FunctionBar()
+  user_motion = TimedRoute(location1,location2,TRANSPORT_SPEEDS["walking"],10)
+  user_motion.create_route()
+  user_motion.write_route("usermotiontestfile.csv")
 """
 
-import sys
+import csv
 from datetime import datetime
-
-import googlemaps
 import pprint
 
 from map_requests import request_directions,request_elevations
-
-#average in meters per second
-WALKING_SPEED = 1.4
+from gps_utils import lla_to_xyz
```

Comment:
For the directory structure, I think adding a top level folder would be helpful so we can tell what imports are from geobeam.